benefits/locale/es/LC_MESSAGES/django.po

```diff
@@ -23,7 +23,7 @@ msgstr "Cierre sesión de Login.gov"
 
 #: benefits/core/migrations/0002_sample_data.py:104
 msgid "eligibility.pages.index.dmv.label"
-msgstr "TODO: Senior Discount Program"
+msgstr "Programa de descuentos para personas mayores"
```

Comment:
Should this be updated to something like.. `eligibility.pages.index.oauth.label`? (This label references the name of the Eligibility option for people who are 65+ with SSN).